cases/management/commands/enrich_ciaa_description.py

```
@@ -0,0 +1,1300 @@
+        if court_number and court_number.upper() not in nums:
+            return (
+                f"title number(s) {sorted(nums)} do not include the special-court "
+                f"number {court_number}"
+            )
+        return None
+
+    # Defendant-headcount patterns the title must avoid: a Devanagari/ASCII
+    # number immediately followed by जना / व्यक्ति / प्रतिवादी (e.g. "१२ जना",
+    # "249 प्रतिवादी"). The court case number itself (080-CR-0098) won't match —
+    # it has no such trailing noun.
+    _HEADCOUNT_RE = re.compile(r"[०-९0-9]+\s*(जना|व्यक्ति|प्रतिवादी)")
```

Ruff (0.15.15)

[warning] String contains ambiguous `०` (DEVANAGARI DIGIT ZERO). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)